tests/Standards/BestIt/Sniffs/SniffTestCase.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace Tests\BestIt\Sniffs;
+
+use PHP_CodeSniffer_File;
+use SlevomatCodingStandard\Sniffs\TestCase as SlevomatTestCase;
+
+/**
+ * Class SniffTestCase
+ *
+ * @package Tests\BestIt\Sniffs
+ *
+ * @author Nick Lubisch <nick.lubisch@bestit-online.de>
+ */
+abstract class SniffTestCase extends SlevomatTestCase
+{
+    /**
+     * Returns the sniff class name.
+     *
+     * @return string sniff class name
+     */
+    protected function getSniffClassName(): string
```

Comment:
We have to talk about this next morning.
Done know the result of the last talk of this topic.

The return type is given by the interface of the Slevomat TestCase